.doc_gen/metadata/serverless_metadata.yaml

```diff
@@ -12,3 +12,5 @@ serverless_S3_Lambda:
             - description: "Consuming an S3 event with Lambda using JavaScript."
               snippet_files:
                 - integration-s3-to-lambda/example.js
+  services:
+    serverless: {S3Lambda}
```

Comment:
But we must have a services stanza - can the bracketed thing be empty?

Reply:
Yes, empty brackets are okay, or just nothing, like:
```
  services:
    serverless:
```
IIRC you need the stanza because that's how the system buckets the example into a service, and the part in the brackets is used only for rendering an API link. If there's no action, you get no link (which is what you want in this case).

Reply:
Sounds like an important detail for the user guide :)